app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -201,6 +210,7 @@ class HomeMenu(
             extensionsItem,
             syncSignInMenuItem,
             accountAuthItem,
+            manageAccountAndDevicesItem,
```

Comment:
If I understand correctly this new menu option should only be shown in the Chinese version of the app.
To avoid it being shown in all apps (if not needed) we can use it as:

```suggestion
            if (Config.channel.isMozillaOnline) manageAccountAndDevicesItem else null,
```